SasquatchMac/SasquatchMac/ViewControllers/AnalyticsViewController.swift

```diff
@@ -25,7 +53,8 @@ class AnalyticsViewController : NSViewController, NSTableViewDataSource, NSTable
     table?.dataSource = self
     NotificationCenter.default.addObserver(self, selector: #selector(self.editingDidBegin), name: .NSControlTextDidBeginEditing, object: nil)
     NotificationCenter.default.addObserver(self, selector: #selector(self.editingDidEnd), name: .NSControlTextDidEndEditing, object: nil)
-  }
+    self.countLabel.stringValue = "Count: \(Int(countSlider.intValue))"
+    }
```

Comment:
Format

Reply:
Pushed the new commit to format code, please check.